meta-mender-core/conf/mender-vars.json

```diff
@@ -40,7 +40,6 @@
     "MENDER_BOOT_PART_SIZE_MB": "",
     "MENDER_BOOT_PART_SIZE_MB_DEFAULT": "",
     "MENDER_CALC_ROOTFS_SIZE": "",
-    "MENDER_CERT_LOCATION": "",
```

Comment:
The ticket mentioned in the commit is wrong, it's MEN-4273, not MEN-4237.

Reply:
Ouch! Thanks